### ISSUE TITLE
Previous work

### ISSUE THREAD

**Issue by pwalsh** [idoivri/israel-municipalities-polygons#1]
Hey,

Have you contacted @kobiluria? https://github.com/kobiluria/MoneyMap, https://github.com/kobiluria/MoneyMap/blob/express/map.geojson and https://github.com/kobiluria/MoneyMap/issues/60

**Comment by idoivri** [comment]
nope. I have the formal polygons, tomorrow they'll be publicly accessible.

Was going to contact kobi, you and a few others once the work I was
finished.

Ido

On Sat, Jun 13, 2015 at 10:20 PM, Paul Walsh notifications@github.com
wrote:

> Hey,
> 
> Have you contacted @kobiluria https://github.com/kobiluria?
> https://github.com/kobiluria/MoneyMap,
> https://github.com/kobiluria/MoneyMap/blob/express/map.geojson and
> kobiluria/MoneyMap#60 https://github.com/kobiluria/MoneyMap/issues/60
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/idoivri/israel-municipalities-polygons/issues/1.

## 

-Ido

"There are 10 types of people in the world: those who understand binary,
and those who don't."
(unknown)

**Comment by kobiluria** [comment]
Hey y'all!!!
How are you all doing!?
Sorry for being out of the loop.
.
Do you want to move some the code to a more active repo ? Do you need
anything from my repo / do you want to sync about what we had?
Thanks , Kobi.

**Comment by pwalsh** [comment]
Hey @kobiluria good to hear from you! I wanted to get your app running for a while, but couldn't, this could help me: https://github.com/kobiluria/MoneyMap/issues/61

@idoivri formal from where exactly? Do you know if we can access historical polygons (I'm kind of obsessed with this in Israel, as things have changed so much over the last half century, that it would be awesome to use historical polygons to visualise it).

**Comment by kobiluria** [comment]
Sure I will look into why it's not running. I'll try to either re-active
myself or at least write the steps to active it.
The auth part was to protect the data. Only if you logged in as a github
user you could do a "post" operation - it seemed to make sense at the time.
I'll try to look at it sometime this week.
The formal polygons dropped or scripted ? If scripted you could push the to
a repo / database with a historical tag.
Thanks,
Kobi

**Comment by idoivri** [comment]
AFAIK, the ministry of interior. Will get more details tomorrow (meeting
"the source"). I took a look; It seems very up to date. I have no
historical stuff though... :-/ But it's going to be a very good practice to
save changes from now on. Any ideas on how @pwalsh ? 

On Sun, Jun 14, 2015 at 4:29 PM, Kobi Luria notifications@github.com
wrote:

> Sure I will look into why it's not running. I'll try to either re-active
> myself or at least write the steps to active it.
> The auth part was to protect the data. Only if you logged in as a github
> user you could do a "post" operation - it seemed to make sense at the time.
> I'll try to look at it sometime this week.
> The formal polygons dropped or scripted ? If scripted you could push the to
> a repo / database with a historical tag.
> Thanks,
> Kobi
> 
> On Sun, Jun 14, 2015 at 02:03 Paul Walsh notifications@github.com wrote:
> 
> > Hey @kobiluria https://github.com/kobiluria good to hear from you! I
> > wanted to get your app running for a while, but couldn't, this could help
> > me: kobiluria/MoneyMap#61
> > https://github.com/kobiluria/MoneyMap/issues/61
> > 
> > @idoivri https://github.com/idoivri formal from where exactly? Do you
> > know if we can access historical polygons (I'm kind of obsessed with this
> > in Israel, as things have changed so much over the last half century,
> > that
> > it would be awesome to use historical polygons to visualise it).
> > 
> > —
> > Reply to this email directly or view it on GitHub
> > <
> > https://github.com/idoivri/israel-municipalities-polygons/issues/1#issuecomment-111786174
> > 
> > .
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/idoivri/israel-municipalities-polygons/issues/1#issuecomment-111827196
> .

## 

-Ido

"There are 10 types of people in the world: those who understand binary,
and those who don't."
(unknown)

**Comment by pwalsh** [comment]
@idoivri if you are meeting them, please ask about this, so we know what they have and what they can/will release. Does the data have polygons for the Palestinian Territories too? Or, at least, Jewish settlements therein? I'm not intending to make this political, but it is very interesting data in the context of mapping and the State.

**Comment by idoivri** [comment]
Yes, sure. updates after tomorrow night (hasadna meetup).

On Sun, Jun 14, 2015 at 5:07 PM, Paul Walsh notifications@github.com
wrote:

> @idoivri https://github.com/idoivri if you are meeting them, please ask
> about this, so we know what they have and what they can/will release. Does
> the data have polygons for the Palestinian Territories too? Or, at least,
> Jewish settlements therein? I'm not intending to make this political, but
> it is very interesting data in the context of mapping and the State.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/idoivri/israel-municipalities-polygons/issues/1#issuecomment-111828909
> .

## 

-Ido

"There are 10 types of people in the world: those who understand binary,
and those who don't."
(unknown)
